swift_package/BrainFlow/Sources/BrainFlowBindings/BrainFlowException.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  BrainFlowException.swift
+//  a Swift binding for BrainFlow's brainflow_exception.h
+//
+//  Created by Scott Miller on 8/23/21.
+//
+
+import Foundation
+
+class BrainFlowException: Error, Equatable {
```

Comment:
lets rename to BrainFlowError for consistency

Reply:
This comment was already resolved.